data/SALD/data.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Salad",
+  "symbol": "SALD",
+  "decimals": 18,
+  "description": "Salad is dedicated to creating earning opportunities on the blockchain by empowering their community through Play2Earn, Learn2Earn and and Decentralized Labor Exchange platforms. The project also aims to support the growth and market penetration of other web3 projects.",
+  "website": "www.salad.ventures",
+  "twitter": "www.x.com/saladventures",
```

Comment:
this should be in https://www.salad.ventures and https://www.x.com/ventures

Reply:
Updated, thank you!